File: python/examples/basics/switch.py
import sys

if len(sys.argv) != 2:
    print("Usage: python switch.py <status_code>")
    sys.exit(1)

status_code = int(sys.argv[1])

match status_code:
    case 401 | 302:
        print("401 or 302")
    case 200:
        print("200")
    case 200:
        print("200 again")
    case _:
        print("other")

File: javascript/src/examples/js/switch.js
"use strict";

var n = 42;

switch (n) {
  case 1: {
    console.log(1);
    break;
  };
  case 42: {
    console.log(42);
  };
  case 'other': {
    console.log('other');
  };
  default: {
    console.log('default')
  };

}

// 42
// other
// default
